arangod/Transaction/Methods.cpp

```diff
@@ -1880,16 +1880,18 @@ Future<OperationResult> transaction::Methods::truncateLocal(std::string const& c
              responses[i].get().statusCode() == fuerte::StatusOK);
         if (!replicationWorked) {
           auto const& followerInfo = collection->followers();
+          LOG_TOPIC("0e2e0", WARN, Logger::REPLICATION)
+              << "truncateLocal: dropping follower " << (*followers)[i]
+              << " for shard" << collection->vocbase().name() << "/" << collectionName
```

Comment:
I suggest to put one more space behind the word "shard" here.

Reply:
Sure, thanks for the hint. I overlooked this initially.